_observing-your-data/query-insights/grouping-top-n-queries.md

```diff
@@ -0,0 +1,331 @@
+              "auto_generate_synonyms_phrase_query": true,
+              "boost": 1.0
+            }
+          }
+        }
+      },
+      "phase_latency_map": {
+        "expand": 0,
+        "query": 15,
+        "fetch": 0
+      },
+      "total_shards": 1,
+      "node_id": "ZbINz1KFS1OPeFmN-n5rdg",
+      "query_hashcode": "484eaabecd13db65216b9e2ff5eee999",
+      "task_resource_usages": [
+        {
+          "action": "indices:data/read/search[phase/query]",
+          "taskId": 64,
+          "parentTaskId": 63,
+          "nodeId": "ZbINz1KFS1OPeFmN-n5rdg",
+          "taskResourceUsage": {
+            "cpu_time_in_nanos": 12161000,
+            "memory_in_bytes": 473456
+          }
+        },
+        {
+          "action": "indices:data/read/search",
+          "taskId": 63,
+          "parentTaskId": -1,
+          "nodeId": "ZbINz1KFS1OPeFmN-n5rdg",
+          "taskResourceUsage": {
+            "cpu_time_in_nanos": 293000,
+            "memory_in_bytes": 3216
+          }
+        }
+      ],
+      "indices": [
+        "my_index"
+      ],
+      "labels": {},
+      "search_type": "query_then_fetch",
+      "measurements": {
+        "latency": {
+          "number": 43,
+          "count": 3,
+          "aggregationType": "AVERAGE"
+        }
+      }
+    }
+  ]
+}
+```
+
+</details>
+
+## Response fields
+
+The response includes the following fields.
+
+Field | Data type        | Description
+:--- |:---| :---
+`top_queries` | Array | The list of top query groups.
+`top_queries.timestamp` | Integer          | The execution timestamp for the first query in the query group.
+`top_queries.source` | Object           | The first query in the query group.
+`top_queries.phase_latency_map` | Object           | The phase latency map for the first query in the query group. The map includes the times in milliseconds the query spent in `expand`, `query`, and `fetch` phases.
+`top_queries.total_shards` | Integer          | The number of shards on which the first query was executed.
+`top_queries.node_id` | String           | The node ID of the node that coordinated the execution of the first query in the query group.
+`top_queries.query_hashcode` | String           | The hashcode that uniquely identifies the query group. This is essentially the hash of the [query structure](#grouping-queries-by-similarity).
+`top_queries.task_resource_usages` | Array of objects | The resource usage breakdown for the various tasks for the first query in the query group.
+`top_queries.indices` | Array | The indexes which the first query in the query group is searching.
+`top_queries.labels` | Object           | Used to label the top query.
+`top_queries.search_type` | String           | The search request execution type (`query_then_fetch` or `dfs_query_then_fetch`). For more information, see the `search_type` parameter in the [Search API documentation]({{site.url}}{{site.baseurl}}/api-reference/search/#url-parameters).
```

Comment:
Should "section" follow "parameter"? The reader can't literally see the parameter.

Reply:
This is referring to a table that lists all parameters, and the `search_type` parameter is one of the rows in that table.